### ISSUE TITLE
Type mismatch for materialProviders

### ISSUE THREAD

**Issue by jodinathan** [dart-lang/angular_components#224]
Just try to add the materialDirectives to the bootstrapStatic provider list like:

```
void main() {
  bootstrapStatic(AppComponent, [
    ac.materialProviders
  ], ngMain.initReflector);
}
```

However, it works if you change it to popupBindings like:

```
bootstrapStatic(AppComponent, [
    ac.popupBindings
  ], ngMain.initReflector);
```

The error: Type 'JSArray<Object>' is not a subtype of type 'List<Provider>'

**Comment by nshahan** [comment]
Are you building with dartdevc? I believe the issue is that dartdevc started type checking lists. I find it still works for now when you build with dart2js.

I can type the materialProviders as List<dynamic> on the next release to fix this. To work around it for now you can change the type in a local version of the package from:

```dart
const List<List<Provider>> materialProviders = const <List<Provider>>[
  popupBindings,
];
```

to:

```dart
const List<dynamic> materialProviders = const [
  popupBindings,
];
```

**Comment by nshahan** [comment]
Fixed in 1eb873c7664b13f63b14b3702dc2196b87607a40.
